demos/python_demos/cocosnet_demo/cocosnet_demo/preprocessing.py

```diff
@@ -0,0 +1,44 @@
+"""
+ Copyright (C) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+import cv2
+
+
+def scatter(source):
+    out = np.zeros((1, 151, 256, 256))
```

Comment:
All of these hardcoded numbers should be replaced with dimensions queried from the model passed by the user.